tests/perf/README.md

```diff
@@ -0,0 +1,15 @@
+# baseUrl default baseurl localhost:3000
+## Can be updated in cypress.config.js or:
+
+yarn cypress run --config baseUrl=http://mybaseurl
```

Comment:
These instructions don't work for me anymore now that there's no `package.json` here. Do you need to set a yarn setting to link to the `frontend/package.json`?

I tried
```
yarn --cwd /kiali/frontend cypress run --config-file .
```
Which seemed like it should work but now cypress is throwing some weird errors.

Reply:
I have tested some options:

- Using --cwd, and it gave some errors. Even creating a new yarn setting, doesn't seems to work. Maybe because is in other path?
- Adding kiali-ui as a depency: Won't install the devDependies, just dependencies. So trying to run the tests, it will raise the error: cypress not found
- Adding a .yarnrc file with:
--modules-folder ../../frontend/node_modules
But fails running cypress, missing typescript dependency. Probably cypress uses a different configuration for its own dependencies? 

I can think of: 
- Keep separating the tests in frontend and tests/perf: Add a package.json in perf dir, with just the two dependencies required (cypress and typescript). It will run faster, but the risk of mismatched dependencies exist (Or look for an alternative like package.json inheritance to get the versions from the frontend package).
- Include perf test into the frontend and create a separate configuration for the /perf tests. Like, use workspaces, https://classic.yarnpkg.com/lang/en/docs/workspaces/ --> It would require to change the /perf location

Reply:
Added a script to install kiali-ui dependencies, updated README.

Reply:
All the different config settings are a pain to get working together. Between cypress, typescript, and yarn there's a lot of different configs to manage. You mind if I open a PR against your branch/repo and you can review it and accept/reject any changes?

Reply:
Please, go ahead!